docs/source/releases/index.rst

```diff
@@ -18,6 +18,14 @@
 Release Notes
 *************
 
+Latest (cutting-edge from GitHub)
+------
+
+.. toctree::
+   :maxdepth: 1
+
+   latest
+
```

Comment:
This actually breaks it - now build_docs.sh automatically u pdates this file.  You shouldn't update it directly.

Reply:
This is fixed.